### DISCUSSION TITLE
Cart Reader PCB Settings

### DISCUSSION THREAD

**Discussion by FoxyMonty** [sanni/cartreader#93] [Need help]
Just wanted to ask and get a confirmation that all of these are the correct settings. It seems like the options have slightly changed from when the 'default settings' example image was taken. 

The specify order number is highlighted on the example image, not sure which option that should be now.

Thanks for any clarification.

![pcb-settings](https://user-images.githubusercontent.com/26324582/107769845-dab7b200-6d38-11eb-9ddd-c0a6ffcab93c.jpg)

**Comment by sanni** [comment]
Remove Order Number:
"No" -> JLCJLCPCB will print the order number somewhere on the board
"Yes" -> JLCJLCPCB will not print the order number on the board but will charge you extra
"Specify a location" -> JLCPCB will print the order number below the SNES slot at no extra cost as I specified in the PCB design file

It's just a cosmetic choice but I would choose "Specify a location".

**Reply by FoxyMonty** [reply]
Thanks for the clarification! I was wondering whether I'd have to specify that location but it's already done, fantastic. Thank you!
